app-server/app/src/test/java/com/codesoom/myseat/controllers/reservations/retrospectives/RetrospectiveUpdateControllerTest.java

```diff
@@ -0,0 +1,154 @@
+package com.codesoom.myseat.controllers.reservations.retrospectives;
+
+import com.codesoom.myseat.domain.Plan;
+import com.codesoom.myseat.domain.Reservation;
+import com.codesoom.myseat.domain.Retrospective;
+import com.codesoom.myseat.domain.User;
+import com.codesoom.myseat.dto.RetrospectiveRequest;
+import com.codesoom.myseat.exceptions.IrrevocableRetrospectiveException;
+import com.codesoom.myseat.exceptions.RetrospectiveNotFoundException;
+import com.codesoom.myseat.services.auth.AuthenticationService;
+import com.codesoom.myseat.services.reservations.ReservationService;
+import com.codesoom.myseat.services.reservations.retrospectives.RetrospectiveUpdateService;
+import com.codesoom.myseat.services.users.UserService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(RetrospectiveUpdateController.class)
+class RetrospectiveUpdateControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    private static final String ACCESS_TOKEN
+            = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaDk";
+
+    private static final String INVALID_TOKEN
+            = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaDk2";
```

Comment:
```suggestion
    private static final String ACCESS_TOKEN
            = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaDk";

    private static final String INVALID_TOKEN = ACCESS_TOKEN + "WRONG_DATA";
```

이렇게 표현하면 더 명확하게 이해될 것 같습니다